src/test/scala/ai/privado/languageEngine/csharp/tagger/sink/CSharpAPITaggerTest.scala

```diff
@@ -0,0 +1,50 @@
+package ai.privado.languageEngine.csharp.tagger.sink
+
+import ai.privado.languageEngine.csharp.CSharpTestBase
+import ai.privado.model.{CatLevelOne, Constants, SourceCodeModel}
+import io.shiftleft.semanticcpg.language.*
+
+class CSharpAPITaggerTest extends CSharpTestBase {
```

Comment:
This test case style is deprecated, use CSharpFrontendTestSuite instead - https://github.com/Privado-Inc/privado-core/blob/main/src/test/scala/ai/privado/testfixtures/CSharpFrontendTestSuite.scala

Reply:
Also to write API, test case you can refer - https://github.com/Privado-Inc/privado-core/blob/main/src/test/scala/ai/privado/languageEngine/java/tagger/sink/api/JavaAPIRetrofitTaggerTest.scala